File: android/app/src/main/java/org/fossasia/openevent/core/bookmark/BookmarkStatus.java
package org.fossasia.openevent.core.bookmark;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class BookmarkStatus {

    private int storedColor;
    private int sessionId;
    private Status actionCode;

    public enum Status {
        CODE_UNDO_ADDED,
        CODE_BLANK,
        CODE_UNDO_REMOVED,
        CODE_ERROR
    }
}

File: android/app/src/main/java/org/fossasia/openevent/core/bookmark/BookmarkStatus.kt
package org.fossasia.openevent.core.bookmark

data class BookmarkStatus(
        var storedColor: Int? = null,
        var sessionId: Int? = null,
        var actionCode: Status? = null) {

    enum class Status {
        CODE_UNDO_ADDED,
        CODE_BLANK,
        CODE_UNDO_REMOVED,
        CODE_ERROR
    }
}
